flink-kubernetes-operator/pom.xml

```diff
@@ -34,6 +34,10 @@ under the License.
     <properties>
         <awaitility.version>4.1.0</awaitility.version>
         <plugins.tmp.dir>${project.build.directory}/plugins</plugins.tmp.dir>
+        <surefire.module.config> <!-- required by
+        FlinkConfigManagerTest
+        -->--add-opens=java.base/java.util=ALL-UNNAMED
+        </surefire.module.config>
```

Comment:
```suggestion
        <!-- required by FlinkConfigManagerTest -->
        <surefire.module.config> --add-opens=java.base/java.util=ALL-UNNAMED </surefire.module.config>
```

Reply:
in case of just one flag it probably makes sense